spec.md

```diff
@@ -1062,7 +1062,7 @@ The mmCIF format allows for so-called micro-heterogeneity on the group-level. Fo
 | elementList          | [Array](#types)   | Array of elements, 0 to 3 character [Strings](#types)            |    Y     |
 | bondAtomList         | [Array](#types)   | Array of bonded atom indices, [Integers](#types)                 |          |
 | bondOrderList        | [Array](#types)   | Array of bond orders as [Integers](#types) between 1 and 4       |          |
-| bondAromaticityList  | [Array](#types)   | Array of bond aromaticity as [Integers](#types) between 0 and 4  |          |
+| bondAromaticityList  | [Array](#types)   | Array of bond aromaticity as [Integers](#types) -1, 1,2,3 and 3  |          |
```

Comment:
Should this read: Array of bond aromaticity as [Integers](#types) 0 or 1

Reply:
Fixed! Thanks!